packages/swap-manager/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@summerfi/swap-manager",
+  "version": "0.0.1",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "scripts": {
+    "tsc": "tsc",
+    "watch": "tsc -w",
+    "test": "jest --setupFiles dotenv/config --silent=false",
+    "testw": "jest --watch",
+    "build": "tsc -b -v tsconfig.build.json",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix"
+  },
+  "dependencies": {
+    "@summerfi/sdk": "workspace:*",
```

Comment:
I don't understand that from the architecture point of view. Why does the `swap-manager` package have a dependency on the `sdk` package? It should be the opposite. The SDK should use the `swap-manager`. I'm afraid there is a circular dependency here.

Reply:
I'm wondering why swap-manager is not part of SDK? is there a reason for it?

Reply:
Discussed offline and decided to add root-level directory for SDK, and then have sdk-common and each manager inside of it. Later on I'll also separate sdk-common into sdk-client and sdk-common. I can explain offline, it will be easier